views/includes/check-your-answers/verification-checks.html

```diff
@@ -1,5 +1,52 @@
 {% import "includes/date-macros.html" as dateMacros %}
 
+{% set ukAgent = "The UK agent" %}
+{% set someoneElse = "Someone from the overseas entity" %}
+
+{% if pageParams.isRegistration %}
```

Comment:
This is a lot of change link Urls. Could we use  the journey base Url "+" CHANGE_SOMETHING to have one set of sets?

Reply:
discussed - look at later